docs/reference/esql/processing-commands/stats.asciidoc

```diff
@@ -1,8 +1,48 @@
 [discrete]
 [[esql-stats-by]]
 === `STATS ... BY`
-Use `STATS ... BY` to group rows according to a common value and calculate one
-or more aggregated values over the grouped rows.
+
+**Syntax**
+
+[source,esql]
+----
+STATS [column1 =] expression1[, ..., [columnN =] expressionN] [BY grouping_column1[, ..., grouping_columnN]]
+----
+
+*Parameters*
+
+`columnX`::
+The name by which the aggregated value is returned. If omitted, the name is
+equal to the corresponding expression (`expressionX`).
+
+`expressionX`::
+An expression that calculates an aggregated value.
```

Comment:
bikeshed comment, but `compute` may be more standard in this context, no?
```suggestion
An expression that computes an aggregated value.
```
However, `calculate` may already be used quite frequently in the rest of the docs, so I'm not sure it's worth splitting hairs now.